content/sql/concepts/operators/terms/null-safe-equal/null-safe-equal.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'NULL SAFE EQUAL'
+Description: 'Performs an inequality comparison similar to the '=' operator, using the logic of an XNOR gate. Can accept NULL as a value and will not output NULL.'
```

Comment:
```suggestion
Description: 'Performs an equality comparison similar to the '=' operator, using the logic of an XNOR gate. Can accept NULL as a value and ensures comparisons involving NULL do not result in NULL.
```
Here I have corrected  `inequality comparison` to `equality comparison` because that is the correct logic for null safe equal operator in SQL.